uaclient/api/u/pro/security/cves/_common/v1.py

```diff
@@ -182,9 +182,6 @@ def get_published_date(self):
         return vulnerability_json_data["published_at"]
```

Comment:
You don't need to make a change, but I suppose the `_common` structure is not needed anymore since there is only one API endpoint.

Reply:
True, I will keep as is just as a reminder that this feature could be extended in the future